ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/tree/member/services/MemberIngestServiceImpl.java

```diff
@@ -23,8 +21,8 @@ public MemberIngestServiceImpl(MemberRepository memberRepository,
 
 	@Override
 	public void addMember(Member member) {
-		Optional<Member> optionalLdesMember = memberRepository.getLdesMemberById(member.getLdesMemberId());
-		if (optionalLdesMember.isEmpty()) {
+		boolean memberExists = memberRepository.memberExists(member.getLdesMemberId());
+		if (!memberExists) {
```

Comment:
if (!memberRepository.memberExists ... ?

Reply:
Mainly did it for readability at first try.. but i'll check again